modules/1plusXRtdProvider.md

```diff
@@ -0,0 +1,76 @@
+# 1plusX Real-time Data Submodule
+
+## Overview
+
+    Module Name: 1plusX Rtd Provider
+    Module Type: Rtd Provider
+    Maintainer: dc-team-1px@triplelift.com
+
+## Description
+
+RTD provider for 1plusX. 
+Enriches the bidding object with Audience & Targeting data
+Contact dc-team-1px@triplelift.com for information.
+
+## Usage
+
+### Build
+```
+gulp build --modules="rtdModule,1plusXRtdProvider,appnexusBidAdapter,..."  
+```
+
+> Note that the global RTD module, `rtdModule`, is a prerequisite of the 1plusX RTD module.
+
+### Configuration
+
+Use `setConfig` to instruct Prebid.js to initilize the 1plusX RTD module, as specified below. 
+
+This module is configured as part of the `realTimeData.dataProviders`
+
+```javascript
+var TIMEOUT = 1000;
+pbjs.setConfig({
+    realTimeData: {
+        auctionDelay: TIMEOUT,
+        dataProviders: [{
+            name: '1plusX',
+            waitForIt: true,
+            params: {
+                customerId: 'acme',
+                bidders: ['appnexus', 'rubicon'],
+                timeout: TIMEOUT
+            }
+        }]
+    }
+});
+```
+
+### Parameters 
+
+| Name              | Type          | Description                                                      | Notes                                                    |
+| :---------------- | :------------ | :--------------------------------------------------------------- |:-------------------------------------------------------- |
+| name              | String        | Real time data module name                                       | Always '1plusX'                                          |
+| waitForIt         | Boolean       | Should be `true` if there's an `auctionDelay` defined (optional) | `false`                                                  |
+| params            | Object        |                                                                  |                                                          |
+| params.customerId | String        | Your 1plusX customer id                                          |                                                          |
+| params.bidders    | Array<string> | List of bidders for which you would like data to be set          | To this date only `appnexus` and `rubicon` are supported |
+| params.timeout    | Integer       | timeout (ms)                                                     | 1000ms                                                   |
+
+## Supported Bidders
+At the moment only Appnexus (`appnexus`) and Magnite (`rubicon`) are supported
```

Comment:
This is a rules violation

Reply:
I just removed the `SUPPORTED_BIDDERS` limitation